src/_pytest/fixtures.py

```diff
@@ -805,7 +855,7 @@ def _teardown_yield_fixture(fixturefunc, it):
         )
 
 
-def _eval_scope_callable(scope_callable, fixture_name, config):
+def _eval_scope_callable(scope_callable, fixture_name: str, config: Config) -> str:
```

Comment:
`scope_callable: Callable[..., str]` - speccing keyword arguments is pretty fiddly, but even without the arguments specified this could catch a fair few issues.

Reply:
I prefer to leave this one for for later.